samples/notifications.js

```diff
@@ -1,168 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
I removed this file, since the samples within it are located in individual files.